source/_integrations/abode.markdown

```diff
@@ -81,8 +81,8 @@ All {% term events %} have the fields:
 | `date` | The date of the event in the format `MM/DD/YYYY`. |
 | `time` | The time of the event in the format `HH:MM AM`. |
 
-There is a unique list of known event_codes that can be found
-[here](https://github.com/MisterWil/abodepy/files/1262019/timeline_events.txt).
+There is a unique list of known event_codes are defined in
```

Comment:
Why is the link to abodepy removed?

Reply:
Aha. I see now that the "groups" is codified as well, so I'll update the docs to link to that.

Reply:
Done in cf6e9d6f6480ce51fe41efb4aabd74e12e71c21e.